src/entities/activity/lib/types/item.ts

```diff
@@ -22,3 +39,52 @@ export type ItemCardButtonsConfig = {
   isSkippable?: boolean
   isNextDisable?: boolean
 }
+
+export type BaseItem<Config, ResponseValues> = {
```

Comment:
It is better to name it just `ActivityItem` then

Reply:
Done